### PR TITLE
Day1-first problem 

### PR DESCRIPTION
**Pull request by takasidk**
Added Day1-problems solutions

### CODE REVIEW

File: CompareTriplets.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+     * The function is expected to return an INTEGER_ARRAY.
+     * The function accepts following parameters:
+     *  1. INTEGER_ARRAY a
+     *  2. INTEGER_ARRAY b
+     */
+
+    public static List<int> compareTriplets(List<int> a, List<int> b)
+    {
+        List<int > arr=new List<int>(){0,0};//this is our list keeps track of a and b                                                    scores 
+        // arr.Add(0);
```
Comment (narayanagunturi) [inline CompareTriplets.cs:30]: remove commented code

File: CompareTriplets.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        // arr.Add(0);
```
Comment (narayanagunturi) [inline CompareTriplets.cs]: remove commented code

File: DiagonalDifference.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+     *
+     * The function is expected to return an INTEGER.
+     * The function accepts 2D_INTEGER_ARRAY arr as parameter.
+     */
+
+    public static int diagonalDifference(List<List<int>> arr)
+    {
+        int leftSum=0;
+        int rightSum=0;
+        int size=arr.Count;    //arr.Count gives us the size of the list(number of rows)
```
Comment (narayanagunturi) [inline DiagonalDifference.cs:29]: No need to use an extra variable for the count. You can use arr.Count in the for loop itself

File: CompareTriplets.cs (hunk 2 above)
Comment (takasidk) [reply CompareTriplets.cs]: Done

File: SimpleArraySum.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    /*
+     * Complete the 'simpleArraySum' function below.
+     *
+     * The function is expected to return an INTEGER.
+     * The function accepts INTEGER_ARRAY ar as parameter.
+     */
+
+    public static int simpleArraySum(List<int> ar)
+    {
+        int sum=0;
```
Comment (narayanagunturi) [inline SimpleArraySum.cs:27]: use var keyword instead of int

File: DiagonalDifference.cs (hunk 3 above)
Comment (takasidk) [reply DiagonalDifference.cs:29]: Okay murthy,I actually declared it so that i could use it multiple times

File: SimpleArraySum.cs (hunk 4 above)
Comment (takasidk) [reply SimpleArraySum.cs:27]: okay murthy

**Pull request merged**